massgov/mayflower
# Update location of JS templates

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

After the update to share assets, the templates directory within the public folder for patternlab was moved. This moves it back.

## Related Issue / Ticket
None

## Steps to Test


1. Build Mayflower patternlab locally
2. Make a change to a file in patternlab/styleguide/source/assets/js/templates/.
3. Confirm that the change is reflected in patternlab/styleguide/public/assets/js/templates/
4. Delete the directory patternlab/styleguide/public/assets/templates/
5. Confirm that pages still load, specifically listing pages like this one after choosing page 2: http://localhost:3000/patterns/05-pages-location-listing/05-pages-location-listing.html

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
